addons/l10n_in_edi/models/account_move_send.py

```diff
@@ -58,5 +58,8 @@ def _call_web_service_before_invoice_pdf_render(self, invoices_data):
                         ),
                         'errors': [error] if not isinstance(error, list) else error,
                     }
+                else:
+                    if invoice.invoice_pdf_report_id:
```

Comment:
```suggestion
                elif invoice.invoice_pdf_report_id:
```
It is almost like it should be the generic behaviour...